schools_change_detection/schools_change_detection.py

```diff
@@ -369,7 +369,7 @@ def generate_update_sql(self, comparison: Comparison) -> str | None:
         """
         if not comparison.attrs:
             return None
-        sql = "UPDATE facilities_lds.nz_facilities\nSET\n"
+        sql = "UPDATE facilities.nz_facilities\nSET\n"
```

Comment:
In the "facilities" schema, the table is "facilities" rather than "nz_facilities".